packages/components/inputs/localized-rich-text-input/src/localized-rich-text-input.tsx

```diff
@@ -17,10 +14,80 @@ import {
 } from '@commercetools-uikit/localized-utils';
 import { LocalizedInputToggle } from '@commercetools-uikit/input-utils';
 import { localized } from '@commercetools-uikit/rich-text-utils';
+import { warning } from '@commercetools-uikit/utils';
 import RichTextInput from './rich-text-input';
 import RequiredValueErrorMessage from './required-value-error-message';
 
-const expandedTranslationsReducer = (state, action) => {
+type TErrors = Record<string, string>;
+type TWarnings = Record<string, ReactNode>;
+
+type TEvent = {
+  target: {
+    id?: string;
+    name?: string;
+    language?: string;
+    value?: string;
+  };
+};
+
+type TLocalizedRichTextInputProps = {
```

Comment:
I'm wondering in which cases we need to add JSDoc comments? If this is not the case we need them, please don't bother

Reply:
updated.

Reply:
Generally it's for generating the component docs in the readme.